interp/constrintern.ml

```diff
@@ -994,9 +994,12 @@ let glob_sort_of_level (level: glob_level) : glob_sort =
   | GType info -> GType [sort_info_of_level_info info]
 
 (* Is it a global reference or a syntactic definition? *)
-let intern_qualid qid intern env ntnvars us args =
+let intern_qualid ?(no_secvar=false) qid intern env ntnvars us args =
```

Comment:
Maybe the parameter should be named `avoid_secvar`?

Reply:
If you want.

Reply:
Not a strong feeling towards it, I was just thinking out loud and suggesting an alternative; so if you like the new one better update the branch, otherwise I'll merge tomorrow.

[Yes, I do agree that part could use a cleanup, also abbreviations could use some love]